### PR TITLE
Issue #38: added UI/UX requirements

### PR DESCRIPTION
**Pull request by ianzachary**
﻿## GitHub Issue

Issue #38

## Description

added section 10, 11 to SRS doc

## How I tested

Compiled locally, used chatgpt for spelling, grammar change suggestions.

### CODE REVIEW

File: docs/SRS-Volere/SRS.tex [hunk 1]
```diff
@@ -136,21 +136,55 @@ \subsection{Functional Requirements}
...
-
+\subsection*{LF-S1 Authority and Trust}
+\textbf{Requirement:} The application appears authoritative and trustworthy.\\
+\textbf{Fit Criterion:} After one encounter with the application, 70 percent of users will agree 
+that they can trust the application and how it handles input data.
 \section{Usability and Humanity Requirements}
 \subsection{Ease of Use Requirements}
-\lips
+\subsection*{UH-EOU1 Human Centered Design (HCD)}
+\textbf{Requirement:} The app interface adheres human-centered design principles to be simple and intuitive to use.\\
```
Comment (emilyperica) [inline docs/SRS-Volere/SRS.tex:158]: requirements should always have 'must' - 'adheres' becomes 'must adhere'

File: docs/SRS-Volere/SRS.tex [hunk 2]
```diff
@@ -136,21 +136,55 @@ \subsection{Functional Requirements}
...
+\textbf{Fit Criterion:} The user interface shall incorporate all four fundamental HCD principles.
 \subsection{Personalization and Internationalization Requirements}
-\lips
+\subsection*{UH-PI1 Language}
+\textbf{Requirement:} The application must provide support for English (US).\\
```
Comment (emilyperica) [inline docs/SRS-Volere/SRS.tex:162]: can we do canadian english :(

File: docs/SRS-Volere/SRS.tex [hunk 3]
```diff
@@ -136,21 +136,55 @@ \subsection{Functional Requirements}
...
+implementation-specific or algorithmic solutions used.
 \subsection{Accessibility Requirements}
-\lips
+\subsection*{UH-A1 Web Content Accessibility Guidelines}
+\textbf{Requirement:} The app must comply with the standardized Web Content Accessibility Guidelines (WCAG).\\
+\textbf{Fit Criterion:} The app must meet or exceed WCAG 2.1 Level AA compliance. This shall be verified through 
+an accessibility audit using automated tools (e.g., WAVE, Axe) and manual reviews, ensuring adherence to guidelines 
+such as text alternatives, keyboard navigation, etc.
+\subsection*{UH-A2 Readability}
+\textbf{Requirement:} All text visible to the user shall be easily readable..\\
```
Comment (emilyperica) [inline docs/SRS-Volere/SRS.tex:186]: trailing '.' here can be removed

File: docs/SRS-Volere/SRS.tex [hunk 4]
```diff
@@ -136,21 +136,55 @@ \subsection{Functional Requirements}
...
+\textbf{Fit Criterion:} The app will meet contrast ratios of at least 4.5:1 for normal text and 3:1 for large text.
```
Comment (emilyperica) [inline docs/SRS-Volere/SRS.tex:187]: why? include a citation here

**Review by emilyperica — CHANGES_REQUESTED** [review]

File: docs/SRS-Volere/SRS.tex (hunk 4 above)
Comment (ianzachary) [reply docs/SRS-Volere/SRS.tex:187]: will actually remove this req. entirely, just realized it's encapsulated by UH-A1

File: docs/SRS-Volere/SRS.tex [hunk 5]
```diff
@@ -137,55 +137,51 @@ \subsection{Functional Requirements}
...
 \subsection*{LF-A1 Discoverability}
-\textbf{Requirement:} Main functionalities of the application interface will have elements that are easily identifiable.\\
+\textbf{Requirement:} Main functionalities of the application interface must have elements that are easily identifiable.\\
 \textbf{Fit Criterion:} Upon viewing the product for the first time, 75 percent of potential 
 users shall be able to locate the interactive element that initiates the score generation process.
 \subsection*{LF-A2 Colour Cohesion}
 \textbf{Requirement:} The application interface must use a consistent color scheme for uniformity.\\
 \textbf{Fit Criterion:} The product shall have color guidelines that specify at least three colors to define a palette 
 (i.e. primary, secondary, and accent colors).
 \subsection*{LF-A3 Resolution}
```
Comment (JaakLipp) [inline docs/SRS-Volere/SRS.tex:147]: For these subsections, could you use \subsubsection{} just to be consistent with mine and appear to fall under a certain requirement number

File: docs/SRS-Volere/SRS.tex [hunk 6]
```diff
@@ -137,55 +137,51 @@ \subsection{Functional Requirements}
...
 \textbf{Fit Criterion:} Users shall be able to easily identify the function of at least 70 percent of interactive elements
 that have unlabeled icons.
 \subsection*{UH-UP2 Information Hiding}
-\textbf{Requirement:} The app shall hide audio processing and transcription details from the user.\\
+\textbf{Requirement:} The app must hide audio processing and transcription details from the user.\\
 \textbf{Fit Criterion:} No user-accessible components of the interface or application shall reveal
 implementation-specific or algorithmic solutions used.
 \subsection{Accessibility Requirements}
 \subsection*{UH-A1 Web Content Accessibility Guidelines}
 \textbf{Requirement:} The app must comply with the standardized Web Content Accessibility Guidelines (WCAG).\\
```
Comment (JaakLipp) [inline docs/SRS-Volere/SRS.tex:181]: could you add an href just to the guidelines we'll follow

File: docs/SRS-Volere/SRS.tex [hunk 7]
```diff
@@ -137,55 +137,51 @@ \subsection{Functional Requirements}
...
 \textbf{Fit Criterion:} The product shall have color guidelines that specify at least three colors to define a palette 
 (i.e. primary, secondary, and accent colors).
 \subsection*{LF-A3 Resolution}
-\textbf{Requirement:} The application will use high definition graphics.\\
+\textbf{Requirement:} The application must use high definition graphics.\\
 \textbf{Fit Criterion:} The application will only display images that are of a resolution of 720p or higher.
 \subsection{Style Requirements}
 \subsection*{LF-S1 Authority and Trust}
-\textbf{Requirement:} The application appears authoritative and trustworthy.\\
+\textbf{Requirement:} The application must appear authoritative and trustworthy.\\
```
Comment (JaakLipp) [inline docs/SRS-Volere/SRS.tex:152]: could you add some justification for why this is a requirement

**Review by JaakLipp — CHANGES_REQUESTED** [review]

**Review by JaakLipp — APPROVED** [review]
LGTM as long as we add sources

**Review by emilyperica — APPROVED** [review]

**Pull request merged**